tests/Application/package.json

```diff
@@ -2,7 +2,7 @@
   "dependencies": {
     "babel-polyfill": "^6.26.0",
     "chart.js": "^2.9.3",
-    "jquery": "^3.4.0",
```

Comment:
Is this backwards compatible? Otherwise we also have to find a way to only enable it conditionally.

Reply:
I have no idea why I upgraded jquery, the Sylius plugin skeleton still has ^3.4.0
Anyway, I reverted the change.